dada2_tutorial_16S.R

```diff
@@ -1,6 +1,6 @@
 #'# dada2 tutorial with MiSeq dataset for Fierer Lab 
-#' *This tutorial created by Angela Oliverio and Hannah Holland-Moritz, and is maintained by current members of the Fierer Lab (Corinne Walsh, Matt Gebert, Kunkun Fan)*     
-#' *Updated March 2nd, 2020*
+#' *This tutorial created by Angela Oliverio and Hannah Holland-Moritz, and is maintained by current members of the Fierer Lab (Corinne Walsh, Matt Gebert, Claire Winfrey, Michael Hoffert)*     
+#' *Updated March 3rd, 2022*
```

Comment:
I think that it'd be useful to add a note specifying the R version we used when updating the tutorial. It could help folks not using the server's version of R to figure out any potential incompatibilities